bluebottle/bluebottle_drf2/serializers.py

```diff
@@ -40,10 +40,10 @@ def to_internal_value(self, data):
 
 class SorlImageField(RestrictedImageField):
     def __init__(self, geometry_string, **kwargs):
-        self.crop = kwargs.pop('crop', 'center')
```

Comment:
Since we do not use kwargs anymore, wouldn't it be cleaner to just change the signature here to:

```
def __init__(self, geometry_string, crop='center', colorspace='RGB'):
```